front/app/components/ConsentManager/index.tsx

```diff
@@ -29,16 +24,13 @@ import {
   getActiveDestinations,
   getCategory,
   categorizeDestinations,
-  getConsentRequired,
+  useConsentRequired,
 } from './utils';
 
 const ConsentManager = () => {
   const [preferences, setPreferences] = useState<IPreferences>({
     functional: true,
   });
-  const [cookieConsent, setCookieConsent] = useState<IConsentCookie | null>(
-    null
-  );
```

Comment:
This caused the modal to flash briefly for users who had already consented.

`cookieConsent` was `null`, so `getConsentRequired` returned `true`. Then `getConsent` was called in the `useEffect` below to set `cookieConsent` in the state, after which `getConsentRequired` returned `false`, and the modal to disappear again.

Reply:
Managing this cookieConsent state ourselves was also not necessary. Ideally, the getConsent calls are all moved into hook versions of the current get functions.